lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -1175,7 +1164,7 @@ def _remove_data_role_from_dedicated_cm_if_needed(  # noqa: C901
         self.opensearch_config.remove_temporary_data_role()
 
         # wait until data moves out completely
-        self.opensearch_exclusions.add_current()
```

Comment:
We should not mix with the voting exclusions here, as we just want to move the shards allocation away.